src/NServiceBus.Hosting.Windows/GenericHost.cs

```diff
@@ -42,15 +41,12 @@ public GenericHost(IConfigureThisEndpoint specifier, string[] args, List<Type> d
             profileManager = new ProfileManager(assembliesToScan, args, defaultProfiles);
         }
 
-        /// <summary>
-        ///     Creates and starts the bus as per the configuration
-        /// </summary>
         public void Start()
         {
             try
             {
                 var startableEndpoint = PerformConfiguration().Result;
-                endpoint =  startableEndpoint.Start().Result;
+                endpoint = startableEndpoint.Start().Result;
```

Comment:
Yes! I would like us to apply that pattern at the outermost level. So in that case if possible we should make Start and Stop async and synchronize higher up the call stack